packages/skeleton/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "@sipe-team/typography",
+  "description": "Typography component for Sipe Design System",
```

```suggestion
  "name": "@sipe-team/skeleton",
  "description": "Skeleton component for Sipe Design System",
```
패키지 이름이랑 설명 바꿔주셔야 할 것 같아요 ㅎㅎ

---

허걱 그대로 사용하다가 딱걸렸군요 하하하 수정하겠습니다 🤣